test/ci-test-docker-selenium.sh

```diff
@@ -13,7 +13,7 @@ main() {
     docker-compose -f selenium-docker-compose.yml run \
         -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
         -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
-        selenium "apt-get update && apt-get -y --no-install-recommends install google-chrome-stable && npm install && npm run bootstrap && ./selenium/bin/deck test -s selenium -u http://rundeck:4440 -h --s3-upload --s3-base ${S3_BASE}" || RET=$?
+        selenium "apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B && apt-get update && apt-get -y --no-install-recommends install google-chrome-stable && npm install && npm run bootstrap && ./selenium/bin/deck test -s selenium -u http://rundeck:4440 -h --s3-upload --s3-base ${S3_BASE}" || RET=$?
```

Comment:
do we need this? maybe the setup_remote_docker fixed the issue, right?

Reply:
The CI implementation of this stage is not the same as Tomcat API Test. And based on my testing in the local environment, if we didn't add the key explicitly it also fails. 
This change is necessary according to  https://askubuntu.com/questions/13065/how-do-i-fix-the-gpg-error-no-pubkey and
https://support.google.com/chrome/thread/4032170/gpg-key-error-debian-repo-for-chrome?hl=en

Reply:
ok